lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -52,8 +52,20 @@ module.exports.prototype = {
                 return;
             }
 
+            var prevToken = file.getPrevToken(comment, {
+                includeComments: true
+            });
+
+            if (prevToken.type === 'Line') {
+                return;
+            }
+
+            if (prevToken.type === 'Punctuator' && prevToken.value === '{') {
```

Comment:
Hmm, your comment on the other PR implies that jQuery would expect the form I put on my last comment. Interesting...